### PR TITLE
Add support for Adafruit Feather RP2040 RFM

### PR DESCRIPTION
**Pull request by timmbogner**
Hope everything looks okay!

### CODE REVIEW

File: src/RadioBoards.h [hunk 1]
```diff
@@ -77,12 +82,15 @@
...
+#elif defined(RADIO_BOARD_FEATHER_RP2040)
+  #include "contributed/Adafruit/Feather_RP2040.h"
+
 #else
   #error "Unsupported or unknown radio board!"
 
 #endif
 
-#if RADIO_BOARDS_DEBUG
+#if defined (RADIO_BOARDS_DEBUG)
```
Comment (jgromes) [inline src/RadioBoards.h]: I propose to keep it as it was - the intention (admittedly undocumented) was to have the macros decide on 0/1, same as `RADIOLIB_DEBUG`.

**Review by jgromes — CHANGES_REQUESTED** [review]
Thank you for the contribution! I left one (minor) comment. Ijn addition, would it also be possible to add `RADIO_BOARD_FEATHER_RP2040` to `keywords.txt`? Keep in mind the file needs true tabs, not spaces.

**Comment by StevenCellist** [comment]
@jgromes how do you envision discerning between 'contributed' and 'maintained'? Would maintained only include boards that are tested by RadioLib contributors? In my own opinion, if users can provide a debug log showing a handful of LoRaWAN uplinks & downlink, that would be OK as well to move a board to maintained, because if something is wrong with the radio, LW won't work. But it quite depends on the definition of 'maintained'.

**Comment by timmbogner** [comment]
@StevenCellist I was also wondering if there was a good method to verify the configuration. I'm not using LoRaWAN, so a different method of proof would work better for me. For this, I only verified that it initialized.

**Comment by jgromes** [comment]
@StevenCellist you are right that it's not a completely clear distinction between the two ... so far I have settled on two criteria for maintained boards:

1. Verified by RadioLib org members
2. Can be quickly tested when users report issues

The first point can be satisfied by a log showing working communication, be it LoRaWAN or P2P (I agree that LoRaWAN OTAA+uplink is a nice end-to-end test). The second one is trickier, because it requires some sort of access to the hardware. So if we move a board from contributed to maintained, and then there's a problem, we might not have a way to replicate the issue.

But again, the distinction is not very clear, we may revisit this separation (or drop it altogether) at a later point.

**Comment by StevenCellist** [comment]
@jgromes sure, looks fair to me. Thread hijack closed ;)

**Review by jgromes — APPROVED** [review]

**Comment by jgromes** [comment]
All looks good now - merged, thank you!

**Pull request merged**